streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamKStreamOuterJoinTest.java

```diff
@@ -884,11 +886,13 @@ public void shouldNotEmitLeftJoinResultForAsymmetricAfterWindow() {
 
             processor.checkAndClearProcessResult();
 
-            // push one item to the first stream; this should produce one full-join item
+            // push one item to the first stream;
+            // this should produce one inner-join item;
+            // and a right-joined item for a3
```

Comment:
We don't produce output for `a3`

Reply:
Removed the line in the comment that says we produce output for a3.